scripts/supervisord/assemble-and-restart

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -eo pipefail
+
+# S2i fails when /tmp/src/ is empty. 
+mkdir -p /tmp/src
+touch /tmp/src/.dummy
+
+# We now run the assembly script. If there is a custom one written in the 
+# source files, we use that instead.
+if [ -f /opt/app-root/src/.s2i/bin/assemble ]; then
```

Comment:
I have obviously missed some discussion about it, but could you please let me know what happened to the java s2i stuff? 

Thanks

Reply:
@geoand What java s2i stuff?

Reply:
@geoand 

> A couple weeks ago I opened a PR against @cdrage 's repo (and said PR got merged) that is the source of this PR, which contained a couple fixes to the bash scripts that made them work with the Java s2i image.
> 
Oh right, I remember now.

> Those changes seem to be gone which is why I am asking :). I am sure that I have just missed some discussion about it, so I just wanted to catch up.

I don't think that you missed something. There was no discussion after we decided that we will use your fix in this PR. 

It is strange that it is not there :-(

Reply:
Thanks @kadel !

@cdrage would you like me to open a new PR against your repo if the changes were made unintentionally?

Reply:
Indeed, those are the changes I had made! Thanks for following up @kadel